src/pages/Components/Notification/index.md

```diff
@@ -3,40 +3,55 @@ title: 'Notification'
 order: 4
 ---
 
-> Definition
+> Notifications are used to bring the user attention about an information, available in four types of alert. A notification often appears as a result of a user action.
 
-## Variations
+The notification pattern is composed of :
+
+- **An icon :** In addition to the frame color, it allows to specify the degree of importance.
+- **A title :** Concise and precise, to quickly guide the user.
+- **A parapgraph of text :** To complete the title to be more precise, or indicate the actions to follow.
+- **A Link (optional):** A link can be used when the message indicate to the user an action to do in a new page.
+
+-- Pattern with the 4 variations preview --
+
+## Four types of alert
+
+> To learn more about the colors usage, please read the [color documentation](/Foundations/Colors/).
+
+### Information
 
-Present the basic anatomy and variations of the component in the top of the page. Should help the user see what it is at
-first glance.
+Informational notifications are **blue**. They are used to highlight a relevant information but not require immediate action.
```

Comment:
Informational notifications are blue. They are used to highlight a piece of relevant information but don't require immediate action.